core/vnl/tests/test_complexify.cxx

```diff
@@ -101,6 +157,16 @@ void test_complexify()
       TEST("vnl_sym_matrix imag",0.0,c_sym_matrix(r,c).imag());
       }
     }
+//  c_sym_matrix
```

Comment:
If this is uncommented, it causes a linker error:

    CMakeFiles/vnl_test_all.dir/test_complexify.cxx.o: In function `test_complexify()':
    test_complexify.cxx:(.text+0x11c7): undefined reference to `vnl_sym_matrix<std::complex<float> >::operator=(vnl_sym_matrix<std::complex<float> > const&)'
    collect2: error: ld returned 1 exit status
    make[2]: *** [core/vnl/tests/vnl_test_all] Error 1
    make[1]: *** [core/vnl/tests/CMakeFiles/vnl_test_all.dir/all] Error 2
    make: *** [all] Error 2